### ISSUE TITLE
[Property 'GAME_CTX' (89253468) in chain string 'DESTROY_SELF(NOT(GAME_CTX.IS_VALID))' doesn't produce a proper property_comp

### ISSUE THREAD

**Issue by WarpQueen** [WarpzoneStudios/Hammerting#8]
From player: 
**Describe the bug**
Callstack and crashdump available here: https://discordapp.com/channels/346642400497762305/771649477601525763/809040388354080800

**To Reproduce**
possibly from trying to equip a handle accidentally?

**Comment by WarpQueen** [comment]
![crash](https://user-images.githubusercontent.com/21261633/107519291-6520da00-6bb0-11eb-983e-f9e5ea83182d.png)
![screenshot](https://user-images.githubusercontent.com/21261633/107519302-68b46100-6bb0-11eb-87ba-1a2b247e1f5f.png)

**Comment by voltagex** [comment]
```
---------------------------
tdtd: The game has crashed.
---------------------------
Error: Property 'GAME_CTX' (89253468) in chain string 'DESTROY_SELF(NOT(GAME_CTX.IS_VALID))' doesn't produce a proper property_comp

Hammerting version 0.2.0.0

Additional assert messages:
entity/ui/popup/right_click_item_menu.entity
DESTROY_SELF(NOT(GAME_CTX.IS_VALID))
GAME_CTX

Callstack:
0: property_component_system_validate_chain_hashes
1: property_component_system_evaluate_chain
2: property_component_system_validate_chain_hashes
3: property_component_system_validate_chain_hashes
4: property_component_system_issue_queries
5: utility_system_set_entity_locked
6: wc_fini
7: wc_fini
8: wc_fini
9: bounding_component_system_segment_intersections_debuglines
10: bounding_component_system_segment_intersections_debuglines
11: thread_rwlock_write_unlock
12: configthreadlocale
13: BaseThreadInitThunk
14: RtlUserThreadStart


Assertion failed: false

File: c:/buildbot/build_tdtd-patch-windows-dev/warpcore/entity/systems/property_component_system.cpp
Line: 340

We would love to see this error message, so we can have a chance to fix the problem. 

If you want to help us with this, please press CTRL+C to copy this text, then send it to bugs@warpzonestudios.com or join us on Discord and show us there: https://discord.gg/pbT9zhw


---------------------------
OK   
---------------------------

```

**Comment by Hericage** [comment]
Fixed in internal builds

**Comment by Srekel** [comment]
Fixed in 0.2.0..1 even :)
